### DISCUSSION TITLE
Scrollable table

### DISCUSSION THREAD

**Discussion by lcmgh** [ratatui/ratatui#146] [Discussion]
Is there something like a scrollable list but with columns and a fixed header?

**Comment by mindoodoo** [comment]
Hmm I don't think so off the top of my head. However it might be an interesting feature to implement, if this is something you'd like to see in `ratatui` please open up an issue requesting the feature ! I think there would be many use cases for this.

**Reply by sayanarijit** [reply]
In addition, @lcmgh, pls provide example code (i.e. how the API could look like) to minimize communication gap.

**Comment by lcmgh** [comment]
I noticed the table widget is already scrollable via keyboard navigation: https://github.com/tui-rs-revival/ratatui/blob/main/examples/table.rs

But it is lacking of a scrollbar.

![image](https://user-images.githubusercontent.com/98088021/233991313-91e49981-e693-475a-81a3-30c48a2d019a.png)
